### PR TITLE
show share ownership percentage

### PR DESCRIPTION
**Pull request by marndt93**
resolves: #6740
image

### CODE REVIEW

**Comment by marndt93** [comment]
Couple questions regarding this PR:
1. should we show percentage for other rows?
2. should this feature only be available for specific games (issue was opened specifically for 18cz which has atypical shares)?

**Comment by roseundy** [comment]
I don't think this should be the default. It needs to be configurable by game.

File: assets/app/view/game/corporation.rb [hunk 1]
```diff
@@ -338,10 +338,13 @@ def entities_rows(entities)
           type = entity.player? ? 'tr.player' : 'tr.corp'
           type += '.bold' if last_acted_upon
           name = entity.player? ? entity.name : "© #{entity.name}"
+          show_percent = @game.instance_variable_get(:@show_share_percent_ownership)
```
Comment (roseundy) [inline assets/app/view/game/corporation.rb:341]: @game.class::SHOW_SHARE_PERCENT_OWNERSHIP

File: lib/engine/game/base.rb [hunk 2]
```diff
@@ -453,6 +456,7 @@ def initialize(names, id: 0, actions: [], at_action: nil, pin: nil, strict: fals
         @round_counter = 0
 
         @optional_rules = init_optional_rules(optional_rules)
+        @show_share_percent_ownership = self.class::SHOW_SHARE_PERCENT_OWNERSHIP
```
Comment (roseundy) [inline lib/engine/game/base.rb:459]: Not needed

File: assets/app/view/game/corporation.rb (hunk 1 above)
Comment (marndt93) [reply assets/app/view/game/corporation.rb:341]: Thanks, couldn't figure out how to access the class property.  kept getting undefined method error...

**Review by tobymao — APPROVED** [review]

**Pull request merged**